def function():
# pass removes error out of terminal
    pass
print("hello")



class User:
    def __init__(self, user_id, username):
        print("ish")
        self.id = user_id
        self.username = username
        self.followers = 0


user_1 = User("001", "angela")


print(user_1.username)
print(user_1.id)
print(user_1.followers)

# user_2 = User()
# user_2.id = "002"
# # reminder: user_2.username is an object  AND  "jack" is an attribute
# user_2.username = "jack"





























